test/DSC.test.ts

```diff
@@ -57,16 +62,19 @@ const setup = async () => {
 
 	return {
 		DSC: dsc,
-		proxy: proxy,
+		Oracle: oracleInstance,
 		instance: instance,
+		proxy: proxy,
+		name: name,
+		symbol: symbol,
 		feeRate: feeRate,
-		owner: owner,
-		users: users,
 		decimals: decimals,
-		minter: owner,
-		feeCollector: owner,
-		Oracle: oracleInstance,
+		feeCollector: treasury,
 		feePrecision: feeprecision,
+		owner: owner,
+		minter: minter,
+		treasury: treasury,
```

Comment:
treasury is duplicate of feeCollector

Reply:
i meant that we have both treasury: `treasury` and `feeCollector: treasury`, but in the tests `feeCollector` is used